recipes/pyepics/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pyepics" %}
+{% set version = "3.2.6" %}
+{% set sha256 = "630ac60bf9362a758211cfd3af4e62c9a0a394af6e5674aababdf6c857ad34a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/pyepics/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+      - python
+      - setuptools
+      - numpy
+  run:
+      - python
+      - numpy
+
+test:
+  imports:
+    - epics
+
+about:
+  home: http://pyepics.github.io/pyepics/
+  license: Epics Open
```

Comment:
I think you need to manually package the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Please copy the url to this PR as a comment in the `meta.yaml`:

https://github.com/pyepics/pyepics/pull/108